exonum-java-binding-cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTx.java

```diff
@@ -78,13 +77,8 @@ public static CreateWalletTx fromMessage(BinaryMessage message) {
   }
 
   @Override
-  public boolean isValid() {
-    return getMessage().verify(CRYPTO_FUNCTION, ownerPublicKey);
-  }
-
-  @Override
-  public void execute(Fork view) {
-    CryptocurrencySchema schema = new CryptocurrencySchema(view);
+  public void execute(TransactionContext context) {
```

Comment:
Question about execute fail handling - right now we just **return** in case of business logic fail, do we need to switch to throwing **TransactionExecutionException** ?

Reply:
https://jira.bf.local/browse/ECR-1994

Reply:
Not at the moment — it is already done in a separate branch https://github.com/exonum/exonum-java-binding/commit/40edd7ebb94422d85e7fd26e13bef757964f1b29